### ISSUE TITLE
single value properties on iframes causes attributes property on JSON object to be incorrect

### ISSUE THREAD

**Issue by felipeballesteros** [HenrikJoreteg/html-parse-stringify#14]
For example:

```
<iframe allowfullscreen frameborder="0" width="698" height="573" scrolling="no" id="someid" title="MailOnline Embed Player" src="http://google.com"></iframe>
```
has a single value property allowfullscreen (no key/value pair) which causes the JSON to output the following properties:

```
"0":"width"
"573":"scrolling"
"698":"height"
"MailOnline Embed Player":"src"
"molvideoplayer":"title"
"no":"id"
allowfullscreen:"frameborder"
```

which of course is incorrect

**Comment by ZingerDerGOD** [comment]
#19 is a potential fix for that issue

**Comment by pconerly** [comment]
fixed via #28
